web/templates/package/index.html.eex

```diff
@@ -59,6 +59,9 @@
         <li role="presentation">
           <a role="menuitem" tabindex="-1" href="<%= package_path(HexWeb.Endpoint, :index, sort_created_params) %>">Recently created</a>
         </li>
+        <li role="presentation">
+          <a role="menuitem" tabindex="-1" href="<%= package_path(HexWeb.Endpoint, :index, sort_updated_params) %>">Recently updated</a>
+        </li>
```

Comment:
Nice. Seems like we do dropped this at some point when rebasing the frontend branch.